public/lang/babele/de/investigator.pathOfCthulhuAbilities.json

```diff
@@ -210,22 +210,22 @@
         },
         "Reassurance": {
             "name": "Beruhigen",
-            "category": "Sozial"
+            "category": "Zwischenmenschlich"
         },
         "Riding": {
             "name": "Reiten",
             "category": "Allgemein"
         },
         "Sanity": {
-            "name": "Geistige Gesundheit",
+            "name": "Verstand",
```

Comment:
Don't agree with this one, though Verstand is shorter, Geistige Gesundheit is more accurate to the concept